src/metrics/captures.ts

```diff
@@ -148,6 +208,22 @@ export class KDRatioMetric implements Metric {
         this.KDAssistsMap[move.uas].kills++;
         this.KDAssistsMap[move.capture.uas].deaths++;
 
+        this.KDAssistsValuesMap[move.capture.uas].deaths++;
+
+        // identify kill values based on captured piece type
+        if (move.capture.type === 'p') {
+          this.KDAssistsValuesMap[move.uas].valueKills++;
+        } else if (move.capture.type === 'n' || move.capture.type === 'b') {
+          this.KDAssistsValuesMap[move.uas].valueKills += 3;
+        } else if (move.capture.type === 'k') {
+          this.KDAssistsValuesMap[move.uas].valueKills += 4; // king's fighting value valued around 4 points in https://en.wikipedia.org/wiki/Chess_piece_relative_value
+        } else if (move.capture.type === 'r') {
+          this.KDAssistsValuesMap[move.uas].valueKills += 5;
+        } else if (move.capture.type === 'q') {
+          this.KDAssistsValuesMap[move.uas].valueKills += 9;
+        } 
```

Comment:
minor but a map of type --> value might be faster than the worst-case 5 boolean checks, and we could add default behavior. A switch statement would apply